src/components/02-typography/lists.njk

```diff
@@ -1,29 +1,96 @@
+<div class="grid-row grid-gap">
+  <div class="mobile-lg:grid-col-4">
+    <h6 class="margin-0">Unordered list</h6>
+    <ul class="usa-list">
+      <li>Unordered list item</li>
+      <li>Unordered list item</li>
+      <li>Unordered list item</li>
+    </ul>
+  </div>
 
-  <div class="grid-row grid-gap">
-    <div class="mobile-lg:grid-col-4">
-      <h6 class="margin-0">Unordered list</h6>
-      <ul class="usa-list">
-        <li>Unordered list item</li>
-        <li>Unordered list item</li>
-        <li>Unordered list item</li>
-      </ul>
-    </div>
+  <div class="mobile-lg:grid-col-4 margin-top-4 mobile-lg:margin-top-0">
+    <h6 class="margin-0">Ordered list</h6>
+    <ol class="usa-list">
+      <li>Ordered list item</li>
+      <li>Ordered list item</li>
+      <li>Ordered list item</li>
+    </ol>
+  </div>
 
-    <div class="mobile-lg:grid-col-4 margin-top-4 mobile-lg:margin-top-0">
-      <h6 class="margin-0">Ordered list</h6>
-      <ol class="usa-list">
-        <li>Ordered list item</li>
-        <li>Ordered list item</li>
-        <li>Ordered list item</li>
-      </ol>
-    </div>
+  <div class="mobile-lg:grid-col-4 margin-top-4 mobile-lg:margin-top-0">
+    <h6 class="margin-0">Unstyled list</h6>
+    <ul class="usa-list usa-list--unstyled">
+      <li>Unstyled list item</li>
+      <li>Unstyled list item</li>
+      <li>Unstyled list item</li>
+    </ul>
+  </div>
+</div>
 
-    <div class="mobile-lg:grid-col-4 margin-top-4 mobile-lg:margin-top-0">
-      <h6 class="margin-0">Unstyled list</h6>
-      <ul class="usa-list usa-list--unstyled">
-        <li>Unstyled list item</li>
-        <li>Unstyled list item</li>
-        <li>Unstyled list item</li>
-      </ul>
+{#Event List Example#}
+
+<div class="grid-row grid-gap margin-top-4 width-mobile">
+  <div class="sds-feed">
```

Comment:
the component name says it's a `feed` component (sds-feed) . Why its in the list section?

Reply:
That's where Christy asked it to be during the meeting on Tuesday, do you want me to put this somewhere else?